tests/settings.py

```diff
@@ -43,6 +43,7 @@
     # Enable allauth.
     'allauth',
     'allauth.account',
+    'allauth.socialaccount',
```

Comment:
It was needed because of the `self.client.get(reverse('account_login'))` in the test. Otherwise it could not render the default template.